lib/api.js

```diff
@@ -48,6 +49,40 @@ function removeCoupling(heroku, app) {
            .then(coupling => deleteCoupling(heroku, coupling.id));
 }
 
+function getPipelineCouplings(heroku, pipelineId) {
+  return heroku.request({
+    method: 'GET',
+    path: `/pipelines/${pipelineId}/pipeline-couplings`,
+    headers: { 'Accept': V3_HEADER }
+  });
+}
+
+function getAppFitler(heroku, appIds) {
+  return heroku.request({
+    method: 'POST',
+    path: `/filters/apps`,
+    headers: { 'Accept': FILTERS_HEADER },
+    body: { in: { id: appIds } }
+  });
+}
+
+function getPipelineApps(heroku, pipelineId) {
```

Comment:
Similar to above:

1. I think we should keep the `Pipeline` infix here, as `Coupling` implies pipeline above.